src/Manager.php

```diff
@@ -229,6 +238,31 @@ public function setBlacklistEnabled($enabled)
         return $this;
     }
 
+    /**
+     * Configuration to set up if show the TokenBlacklistedException
+     * can be throwable or not.
+     *
+     * @param bool $showBlackListException
+     *
+     * @removed this
+     */
+    public function setBlackListExceptionEnabled($showBlackListException = true)
+    {
+        $this->showBlackListException = $showBlackListException;
+
+        return $this;
+    }
+
+    /**
+     * Get if the blacklist instance is enabled.
+     *
+     * @return bool
+     */
+    public function gettBlackListExceptionEnabled()
```

Comment:
Here is the defintion with the additional t

Reply:
Solved